govcd/extension.go

```diff
@@ -116,3 +116,15 @@ func getExtension(vcdClient *VCDClient) (*types.Extension, error) {
 
 	return extensions, nil
 }
+
+func queryVirtualCenters(vdcCli *VCDClient, filter string) ([]*types.QueryResultVirtualCenterRecordType, error) {
```

Comment:
This function should be public.

Reply:
Please also put a comment above public function explaining what it does.